src/Smidge.Core/BundleFileSetGenerator.cs

```diff
@@ -50,15 +49,14 @@ public IEnumerable<IWebFile> GetOrderedFileSet(Bundle bundle, PreProcessPipeline
         /// <returns></returns>
         public IEnumerable<IWebFile> GetOrderedFileSet(IEnumerable<IWebFile> files, PreProcessPipeline pipeline)
         {
-            var customOrdered = new HashSet<IWebFile>(WebFilePairEqualityComparer.Instance);
-            var defaultOrdered = new HashSet<IWebFile>(WebFilePairEqualityComparer.Instance);
+            var customOrdered = new List<IWebFile>(1);
```

Comment:
What is the reason this is set to a capacity of 1?

Reply:
No reason, it shouldn't have been there 🙂 Although it wouldn't be a bad idea to set both to the capacity of files, but then files should probably be a collection.